### ISSUE TITLE
Centralisation

### ISSUE THREAD

**Issue by Y0ursTruly** [Y0ursTruly/slate#1]
Is it really peer to peer if I'm routing traffic through ngrok?
Great, they can't see the data you're sending, but you can be anywhere in the world and if ngrok is down, oof!
#
I decided since the start of this year that a lot of changes are going to come to this repository, turning _this repo here_ into the server u can setup to make your own "whiteboard" for users who connect to see the "contact info" of others and establish WebRTC peer to peer connections
#
In short, I'm getting serious about this project and am saying something because I see this repository is still watched 😳
